.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-shortcut-reference-link] Use the trailing [] on reference links](https://app.codacy.com/manual/kamiazya/ts-graphviz/pullRequest?prid=4784559)